sources/packages/backend/workflow/test/2022-2023/fulltime-assessment/fulltime-assessment-2022-2023.e2e-spec.ts

```diff
@@ -0,0 +1,122 @@
+
+describe(`E2E Test Workflow  fulltime-assessment-${PROGRAM_YEAR}`, () => {
+  let zeebeClientProvider: ZBClient;
+  beforeAll(async () => {
+    zeebeClientProvider = new ZBClient();
+  });
+
+  it("Should generate expected fulltime assessment values for a single and independent student.", async () => {
+    const assessmentConsolidatedData = getFakeAssessmentConsolidatedData(
+      OfferingIntensity.fullTime,
+      PROGRAM_YEAR,
+      "2023-02-01",
+      "2023-05-24",
+    );
+    const expectedAssessmentData: AssessmentModel = {
+      weeks: assessmentConsolidatedData.offeringWeeks,
+      tuitionCost:
+        assessmentConsolidatedData.offeringActualTuitionCosts +
+        assessmentConsolidatedData.offeringMandatoryFees,
+      childcareCost: 0,
+      livingAllowance: 1672,
+      totalAssessedCost: 27672,
+      totalFamilyIncome: 40000,
+      totalFederalAward: 5600,
+      otherAllowableCost: 0,
+      transportationCost: 0,
+      secondResidenceCost: 0,
+      totalAssessmentNeed: 55049.19807692308,
+      booksAndSuppliesCost: 5000,
+      totalProvincialAward: 11037.417692307692,
+      alimonyOrChildSupport: 0,
+      federalAssessmentNeed: 26000,
+      exceptionalEducationCost:
+        assessmentConsolidatedData.offeringExceptionalExpenses,
+      provincialAssessmentNeed: 27593.544230769232,
+      parentAssessedContribution: null,
+      partnerAssessedContribution: null,
+      studentTotalFederalContribution: 216.34615384615384,
+      studentTotalProvincialContribution: 78.45576923076923,
+    } as AssessmentModel;
+
+    const calculatedAssessment =
+      await zeebeClientProvider.createProcessInstanceWithResult<
+        AssessmentConsolidatedData,
+        CalculatedAssessmentModel
+      >(`fulltime-assessment-${PROGRAM_YEAR}`, assessmentConsolidatedData);
+    // TODO: totalFederalContribution and totalProvincialContribution needs to be validated
+    // once it is fixed in bpmn.
+    expect(calculatedAssessment.variables.offeringWeeks).toBe(
+      expectedAssessmentData.weeks,
+    );
+    expect(calculatedAssessment.variables.calculatedDataTotalTutionCost).toBe(
+      expectedAssessmentData.tuitionCost,
+    );
+    expect(calculatedAssessment.variables.calculatedDataChildCareCost).toBe(
+      expectedAssessmentData.childcareCost,
+    );
+    expect(
+      calculatedAssessment.variables.calculatedDataTotalMSOLAllowance,
+    ).toBe(expectedAssessmentData.livingAllowance);
+    expect(calculatedAssessment.variables.calculatedDataTotalCosts).toBe(
+      expectedAssessmentData.totalAssessedCost,
+    );
+    expect(calculatedAssessment.variables.calculatedDataTotalFamilyIncome).toBe(
+      expectedAssessmentData.totalFamilyIncome,
+    );
+    expect(calculatedAssessment.variables.awardNetFederalTotalAward).toBe(
+      expectedAssessmentData.totalFederalAward,
+    );
+    expect(calculatedAssessment.variables.calculatedDataChildCareCost).toBe(
```

Comment:
you are already checking `calculatedDataChildCareCost` in line 64. is this `calculatedDataChildCareCost`?  is it the same as `otherAllowableCost`?

Reply:
Yes that is because calculatedDataChildCareCost is used to map 2 outputs as per workflow 
![image](https://user-images.githubusercontent.com/54600590/225157032-5a9c3425-3787-47af-9313-661df7cb8207.png)